.env

```diff
@@ -0,0 +1,10 @@
+# Environment variables declared in this file are automatically made available to Prisma.
+# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB (Preview) and CockroachDB (Preview).
+# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+
+NODE_ENV="development"
+APP_SECRET="jwtsecret123"
+NODE_PATH="./src"
+DATABASE_URL="postgres://fndnziax:YEbewgjpbkceCXlmN46uXIYFuMPrkpbs@tyke.db.elephantsql.com/fndnziax"
```

Comment:
.env files should NEVER be committed to a public repo. This is a security concern and I need you to add it to the `.gitignore`. Also you'll need to change the APP_SECRET and probably update the DB password.

Reply:
"NODE_ENV", "APP_SECRET"  and "NODE_PATH" were added while following an issue with database connectivity. They will be removed now.